src/styles/blocks/stars.css

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+
+  margin-right: 4px;
+
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
you can use shorted version 

 background: url(../../images/star.svg) no-repeat center;